patches/tModLoader/Terraria/ID/SetFactory.TML.cs

```diff
@@ -119,8 +135,16 @@ public void RegisterNamedCustomSet<T>(string key, T defaultValue, ref T[] input)
 					value[i] = input[i];
 				}
 			}
+
+			if(ModCompile.activelyModding)
+				Logging.tML.Info($"Custom Set '{key}' (Type: {typeof(T).Name}) is merging with additional data");
```

Comment:
We want to log the mods which are using the set, perhaps only when there's 2 or more of them?

Reply:
Currently this is buggy due to the static initializer running twice issue.

Reply:
Made some more fixes here, only logs when actual changes happen.